ftrackhq/ftrack-python
# ci: Set up automatic publish on tag push for python-api

Partially Resolves FT-faedda3e-64d7-11ed-86b3-42bf04dc3420 and FT-d7873d26-5eb3-11ed-8b21-3a11a67cec34


## Changes


This PR sets up the build and publish step of the new CI release flow for ftrack-python.

<img width="387" alt="image" src="https://user-images.githubusercontent.com/2225828/203809292-1408a25a-0214-435e-ad0b-169f26b6bb91.png">


## Test



- Make sure the build was uploaded correctly to the test registry.
- Check out a working build towards the test registry here https://github.com/ftrackhq/ftrack-python/actions/runs/3541916184/jobs/5946873225
- Check out the published test here https://test.pypi.org/project/ftrack-python-api/0.0.1b2/
- Look at the pipeline and make sure you understand it
